examples/picow_demo.py

```diff
@@ -88,7 +88,7 @@
         )
 
         if response["rowcount"] == 1:
-            print(f"Average temperature over last 24hrs: {response["rows"][0][0]}")
+            print(f"Average temperature over last 24hrs: {response['rows'][0][0]}")
             num_iterations = 0
```

Comment:
That's a [syntax error at least on CPython 3.7](https://github.com/crate/micropython-cratedb/actions/runs/11599458674/job/32297630588?pr=25#step:7:60). I wonder why it is apparently not the same on either MicroPython or CPython 3.13.